Examples/Io/Performance/ActsExamples/Io/Performance/CKFPerformanceWriter.hpp

```diff
@@ -108,15 +113,18 @@ class CKFPerformanceWriter final : public WriterT<ConstTrackContainer> {
   TrackSummaryPlotTool m_trackSummaryPlotTool;
   TrackSummaryPlotTool::TrackSummaryPlotCache m_trackSummaryPlotCache{};
 
+  /// For optional output of the matching details
+  TTree* m_matchingTree{nullptr};
```

Comment:
this will always be `nullptr` right now, no?

Reply:
Ah I messed up the `git checkout -p`...